src/etcd/etcd_utils.h

```diff
@@ -0,0 +1,20 @@
+
+#ifndef CREDIS_UTILS_H
+#define CREDIS_UTILS_H
+
+#include <vector>
+#include "etcd3/include/etcd3.h"
+
+namespace utils {
```

Comment:
How do you feel about not using namespace for this PR?  Reason is we don't have a top-level `namespace credis`.  If we do, having a `namespace utils` makes a lot of sense.